jujugui/static/gui/src/app/components/entity-details/header/header.js

```diff
@@ -46,8 +46,17 @@ class EntityHeader extends React.Component {
       // format the method expects.
       this.props.deployService(entityModel, undefined, plans, activePlan);
     } else {
-      var id = entity.id.replace('cs:', '');
-      this.props.getBundleYAML(id, this._getBundleYAMLCallback.bind(this));
+      const id = entity.id.replace('cs:', '');
+      let bundleUrl;
+      try {
```

Comment:
So we don't know whether this is a legacy string or not? I think this should be an information that we have, without having to guess.